packages/element-web-module-api/src/api/i18n.ts

```diff
@@ -49,4 +49,10 @@ export interface I18nApi {
      * @param variables - Optional variables to interpolate into the translation
      */
     translate(key: keyof Translations, variables?: Variables): string;
+
+    /**
+     * Convert a timestamp into a human-readable time string
+     * @param timeMillis - The time in milliseconds since epoch
+     */
```

Comment:
Not sure this really offers enough description for me to know what it does if I was to call it, some notes on the human readable format would be good

Reply:
Added. In practice in the impl, it's always a relative time, but I don't know if the doc should be this specific as in practice I feel like it would be reasonable for it to decide that in some circumstances it's better to just display an absolute time.